example/service/s3/sync/sync.go

```diff
@@ -69,10 +70,19 @@ func (iter *SyncFolderIterator) UploadObject() s3manager.BatchUploadObject {
 		iter.err = err
 	}
 
+	fileSplit := strings.Split(fi.key, ".")
+	extension := "." + fileSplit[len(fileSplit)-1]
+	mimeType := mime.TypeByExtension(extension)
+
+	if mimeType == "" {
+		mimeType = "binary/octet-stream"
+	}
+
 	input := s3manager.UploadInput{
-		Bucket: &iter.bucket,
-		Key:    &fi.key,
-		Body:   body,
+		Bucket:      &iter.bucket,
+		Key:         &fi.key,
+		Body:        body,
+		ContentType: aws.String(mimeType),
```

Comment:
nit for consistency with rest of input parameters this should just take the address of `mimeType`, instead of `aws.String`

```go
ContentType: &mimeType,
```